pandas/tests/test_categorical.py

```diff
@@ -871,10 +879,12 @@ def test_rename_categories(self):
         res = cat.rename_categories([1, 2, 3])
         self.assert_numpy_array_equal(res.__array__(), np.array([1, 2, 3, 1]))
         self.assert_numpy_array_equal(res.categories, np.array([1, 2, 3]))
-        self.assert_numpy_array_equal(cat.__array__(),
-                                      np.array(["a", "b", "c", "a"]))
-        self.assert_numpy_array_equal(cat.categories,
-                                      np.array(["a", "b", "c"]))
+
+        exp_cat = np.array(["a", "b", "c", "a"], dtype=np.object_)
+        self.assert_numpy_array_equal(cat.__array__(), exp_cat)
```

Comment:
can do in a followup, but I suspect some of these tests would be more clear to construct the expected ``Categorical`` and use ``assert_equal_categorical`` (when these were originally written we didn't hav e it).